src/topology/README.md

```diff
@@ -95,27 +95,27 @@ const toplogy = new Topology({
 
 #### Set a peer
 
-- `topology.peers.set(id, peerInfo)`
+- `topology.peers.set(id, peerId)`
 
 Add a peer to the topology.
 
 **Parameters**
 - `id` is the `string` that identifies the peer to add.
-- `peerInfo` is the [PeerInfo][peer-info] of the peer to add.
+- `peerId` is the [PeerId][peer-id] of the peer to add.
 
 #### Notify about a peer disconnected event
 
-- `topology.disconnect(peerInfo)`
+- `topology.disconnect(peerId)`
 
 **Parameters**
-- `peerInfo` is the [PeerInfo][peer-info] of the peer disconnected.
+- `peerId` is the [PeerIid][peer-id] of the peer disconnected.
```

Comment:
```suggestion
- `peerId` is the [PeerId][peer-id] of the peer disconnected.
```

Reply:
We're apparently missing the ref link for these at the bottom of the readme, we need to add that.